modules/gitops-logging-into-keycloak.adoc

```diff
@@ -0,0 +1,53 @@
+[id="logging-into-keycloak_{context}"]
+= Logging into Keycloak
+
+.Procedure
+
+. After creating the Argo CD instance, login into the keycloak console.
++
+[source,yaml]
+----
+kubectl -n argocd get route keycloak
+
+NAME   HOST/PORT                                                                PATH   SERVICES   PORT    TERMINATION   WILDCARD
+keycloak    keycloak-default.apps.ci-ln-******.origin-ci-int-aws.dev.**.com          keycloak        <all>   reencrypt     None
+----
+. Get the Keycloak credentials which is stored as environment variables in the keycloak pod.
```

Comment:
Once again, you capitalized `Keycloak Pod` earlier, so please do so here. 

Unless there's a difference between Keycloak Pod and keycloak pod?